File: app/src/main/java/info/mx/tracks/tools/IAddMobHelper.java
package info.mx.tracks.tools;

import android.app.Activity;

public interface IAddMobHelper {

    void onCreateView(Activity activity);

    void onPause(Activity activity);

    void onResume(Activity activity);

    void onDestroy(Activity activity);

    void showInterstitial(Activity activity);
}

File: app/src/main/java/info/mx/tracks/tools/IAddMobHelper.kt
package info.mx.tracks.tools

import android.app.Activity

interface IAddMobHelper {
    fun onCreateView(activity: Activity)

    fun onPause(activity: Activity)

    fun onResume(activity: Activity)

    fun onDestroy(activity: Activity)

    fun showInterstitial(activity: Activity)
}
